styleguide/_themes/derek/scss/components/tabs.scss

```diff
@@ -1,94 +1,111 @@
-.nav-tabs {
-  border: 0;
+.rsTabs {
+  border-bottom: 0;
   margin-bottom: 0;
-  text-align: center;
+}
 
-  & > li {
-    background-color: $teal-dark;
-    border-top: 5px solid $teal-dark;
-    display: inline-block;
-    float: none;
-    padding: 0;
+.nav-tabs-li {
+  border: 1px solid $gray-lighter;
+  display: inline-block;
+  margin-bottom: 0;
+  padding: 0;
+  top: 3px;
 
-    &:hover {
-      border-top: 5px solid $white;
+  &:first-child {
+    margin-left: 10px;
+  }
 
-      a {
-        background-color: $teal-dark;
-      }
-    }
+  & > .nav-tabs-a {
```

Comment:
`.rsTabs-tabLink`?

Also, does this have to be nested here, or can it just be placed in its own top-level item?   I'm assuming its because of some weird bootstrap `li > a` declarations, but just trying to make sure.